src/latexgen.cpp

```diff
@@ -367,7 +368,12 @@ static void writeLatexMakefile()
       << "\t      latex_count=`expr $$latex_count - 1` ;\\" << endl
       << "\t    done" << endl
       << "\t" << mkidx_command << " refman.idx" << endl
-      << "\t$(LATEX_CMD) refman" << endl << endl;
+      << "\t$(LATEX_CMD) refman" << endl;
+    if(!pdf_dst_file.isEmpty())
+    {
+      t << "\tcp refman.pdf " << pdf_dst_file << endl;
+    }
+    t << endl;
```

Comment:
And the native Windows version (`writeMakeBat`) ?

Why copy the file and not rename / move?

Reply:
I was not sure, if the standard behaviour should be kept as much as possible. If any other software relies on finding a refman.pdf in the folder, it would break.

Reply:
The user requests the change so he is also responsible for "oher" software. Doxygen does not rely on it (as far as I know).

Reply:
Simply missed it. I'm using the build from sources on MSYS2 and my make.bat is not generated.